app/services/manage_action.rb

```diff
@@ -24,5 +29,15 @@ def action_user
   def page
     @page ||= Page.find(@params[:page_id])
   end
+
+  def post_to_queue
+    queue_message = {
+      type: 'action',
+      params: {
+        slug: page.slug,
+        email: action_user.email}.merge(@params)}
```

Comment:
Minor nitpick, but moving the closing `}` to a new line feels a lot more readable to me.